harness/determined/util.py

```diff
@@ -330,6 +330,20 @@ def legacy_trial_entrypoint_to_script(trial_entrypoint: str) -> List[str]:
     return ["python3", "-m", "determined.exec.harness", trial_entrypoint]
 
 
+def force_create_symlink(src: str, dst: str) -> None:
+    os.makedirs(src, exist_ok=True)
+    try:
+        os.symlink(src, dst, target_is_directory=True)
+    except FileExistsError:
+        if os.path.islink(dst):
+            os.unlink(dst)
+        elif os.path.isfile(dst):
+            os.unlink(dst)
+        else:
+            shutil.rmtree(dst)
+        os.symlink(src, dst, target_is_directory=True)
```

Comment:
maybe consider a try/except:pass on the second one?  Because if two workers start at the same time, it's ok if one of the second time.